io.github.hakuneko.HakuNeko.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id": "io.github.hakuneko.HakuNeko",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "hakuneko",
+    "separate-locales": false,
+    "rename-desktop-file": "hakuneko-desktop.desktop",
+    "rename-icon": "hakuneko-desktop",
+    "finish-args": [
+		"--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+		"--share=network",
+		"--device=dri",
+        "--filesystem=home",
+	],
+	"modules": [
+		{
+			"name": "hakuneko-desktop",
+			"buildsystem": "simple",
+			"build-commands": [
+				"ar x hakuneko-desktop_*.deb",
+				"rm -f hakuneko-desktop_*.deb",
+				"tar xf data.tar.xz",
+				"rm -f control.tar.gz data.tar.xz debian-binary",
+				"cp -r usr/* opt/* /app",
+				"chmod -R a-s,go+rX,go-w /app/lib/hakuneko-desktop/hakuneko",
+				"rm -rf /app/share/icons/hicolor/1024x1024/",
+				"install hakuneko.sh /app/bin/hakuneko",
+				"install -Dm644 io.github.hakuneko.HakuNeko.xml /app/share/appdata/io.github.hakuneko.HakuNeko.xml"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"only-arches": ["x86_64"],
+					"url": "https://github.com/manga-download/hakuneko/releases/download/v6.1.7/hakuneko-desktop_6.1.7_linux_amd64.deb"
+					"sha256": "9ce203908ee50c49e7fd16e3b7c3fac2eeb23b51d1e991310ee7e5b56416da93"
+				},
+				{
+					"type": "script",
+					"dest-filename": "hakuneko.sh",
+					"commands": [
+						"exec env TMPDIR=$XDG_CACHE_HOME /app/lib/hakuneko-desktop/hakuneko \"$@\""
+						    ]
+                },
+                {
+                    "type": "file",
+                    "path": "io.github.hakuneko.HakuNeko.xml"
```

Comment:
The file should have a suffix of `.metainfo.xml`

Reply:
I added the metainfo suffix onto path as you suggested.